draft-ietf-secevent-token.txt

```diff
@@ -486,95 +542,101 @@ Internet-Draft          draft-ietf-secevent-token          February 2018
+      representing a user resource that was modified.  A token
+      identifier for a revoked token.  If used, the profiling
+      specification SHOULD define the content and format semantics for
+      the value.  This claim is OPTIONAL, as the principal for any given
+      profile may already be identified without the inclusion of a
+      subject claim.  Note that some SET profiles MAY choose to convey
+      event subject information in the event payload (either using the
+      "sub" member name or another name), particularly if the subject
+      information is relative to issuer information that is also
+      conveyed in the event payload, which may be the case for some
+      identity SET profiles.
+
+   "exp" (Expiration Time) Claim
+      As defined by Section 4.1.4 of [RFC7519], this claim is the time
+      after which the JWT MUST NOT be accepted for processing.  In the
+      context of a SET however, this notion does not typically apply,
+      since a SET represents something that has already occurred and is
+      historical in nature.  Therefore, its use is NOT RECOMMENDED.
+      (Also, see Section 4.5 for additional reasons not to use the "exp"
+      claim in some SET use cases.)
 
    The following new claims are defined by this specification:
 
-   events
+   "events" (Security Events) Claim
       This claim contains a set of event statements that each provide
       information describing a single logical event that has occurred
-      about a Security Subject (e.g., a state change to the subject).
-      Multiple event identifiers with the same value MUST NOT be used.
-      The "events" claim SHOULD NOT be used to express multiple
-      independent logical events.
-
-      The value of the "events" claim is a JSON object whose members are
-      name/value pairs whose names are URIs identifying the event
+      about a security subject (e.g., a state change to the subject).
 
 
 
-Hunt, et al.             Expires August 6, 2018                [Page 10]
+Hunt, et al.            Expires September 1, 2018              [Page 11]
 
 Internet-Draft          draft-ietf-secevent-token          February 2018
 
 
+      Multiple event identifiers with the same value MUST NOT be used.
+      The "events" claim SHOULD NOT be used to express multiple
+      independent logical events.
+
+      The value of the "events" claim is a JSON object whose members are
+      name/value pairs whose names are URIs identifying the event
       statements being expressed.  Event identifiers SHOULD be stable
       values (e.g., a permanent URL for an event specification).  For
       each name present, the corresponding value MUST be a JSON object.
       The JSON object MAY be an empty object ("{}"), or it MAY be a JSON
-      object containing data described by the Profiling Specification.
+      object containing data described by the profiling specification.
 
-   txn
+   "txn" (Transaction Identifier) Claim
```

Comment:
I wonder if Yaron's concerns might be better addressed if it said "in which multiple related JWTs or SETs are issued".  I can go either way...you choose.